restore-sql-server-transaction-logs/scheduled-upload/upload-script.ps1

```diff
@@ -0,0 +1,46 @@
+$settingsFile="<full-path-to-your-script>\settings.json"
+$uploadedObjectsNumber=0
+$currentDateTimeUtc = (Get-Date).ToUniversalTime()
+
+# read from json settings file
+$settings = Get-Content $settingsFile -Raw | ConvertFrom-Json
```

Comment:
I don't see any validation. What happens if a user writes a valid JSON with the right keys, but wrong values?

Reply:
Indeed, it was missing. Added validations.

Reply:
LGTM without complicating things too much